src/search/ui/offer.controller.js

```diff
@@ -234,6 +234,10 @@ function OfferController(
   };
 
   controller.showButtonConfirmed = function () {
+    if (_.get(appConfig, 'hideButtonConfirmed') === true) {
+      return false;
+    }
+
```

Comment:
Did that confirm button do anything that we should now maybe do automatically if the button is hidden?

Reply:
It added the label `bevestigd2020`, 

The ticket states:
`With the introduction of offerStatus there is no need to keep this button along the frontend`

So I understand this to mean that because of offerStatus we don't have to use the `bevestigd` label anymore

Reply:
Makes sense!